cli/src/main/scala-2.11/coursier/cli/Bootstrap.scala

```diff
@@ -119,32 +119,25 @@ final case class Bootstrap(
       }
 
     val (urls, files) =
-      if (options.standalone)
-        (
-          Seq.empty[String],
-          helper.fetch(
-            sources = false,
-            javadoc = false,
-            artifactTypes = artifactOptions.artifactTypes(sources = false, javadoc = false)
-          )
-        )
-      else
-        (
-          helper.artifacts(
-            sources = false,
-            javadoc = false,
-            artifactTypes = artifactOptions.artifactTypes(sources = false, javadoc = false)
-          ).map(_.url),
-          Seq.empty[File]
-        )
+      helper.fetchMap(
+        sources = false,
+        javadoc = false,
+        artifactTypes = artifactOptions.artifactTypes(sources = false, javadoc = false)
+      ).toList.foldLeft((List.empty[String], List.empty[File])){
+        case ((urls, files), (url, file)) =>
+          if (options.standalone) (urls, file :: files)
+          else url match {
```

Comment:
Nitpicking: that `match` doesn't really match anything, it should just be a bunch of `if` / `else`.